File: develop/binder-canary-sample/src/main/java/com/example/binder/canary/sample/MainActivity.java
package com.example.binder.canary.sample;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.zipper.develop.binder.canary.BinderCanary;
import com.zipper.develop.binder.canary.MonitorConfig;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        BinderCanary.init(new MonitorConfig());
        BinderCanary.monitor();
    }
}

File: lib-aspectj-ipc/sample/src/main/java/com/zipper/lib/aspectj/ipc/sample/MainActivity.kt
package com.zipper.lib.aspectj.ipc.sample

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.zipper.lib.aspectj.ipc.IIpcLog

class MainActivity : AppCompatActivity() , IIpcLog {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        log("MainActivity#onCreate")
    }

    fun dump2(view: View) {
        log("dump 2")
        val intent = Intent(this, Process2MainActivity::class.java)
        startActivity(intent)
    }
}